Batch mode (#288)
Qiskit/documentation: docs/run/sessions.mdx

# About sessions

> A session allows a collection of jobs to be grouped and jointly scheduled by the Qiskit Runtime service, facilitating iterative use of quantum computers without incurring queuing delays on each iteration. This eliminates artificial delays caused by other users’ jobs running on the same quantum device during the session. If you want to submit multiple jobs at once, use the [`Batch`](run-primitives-batch) class instead.

Reviewer: ```suggestion
A session allows a collection of jobs to be grouped and jointly scheduled by the Qiskit Runtime service, facilitating iterative use of quantum computers without incurring queuing delays on each iteration. This eliminates artificial delays caused by other users’ jobs running on the same quantum device during the session. If you don't need iterative processing and can send all jobs at once, use the [`Batch`](run-primitives-batch) class instead.
```
You can still send multiple jobs under `Session`.

    -   No further jobs can be submitted to the session.
    -   The session cannot be reopened.

> Sessions run _iteratively_.  To run jobs in batch mode, see [batch](run-primitives-batch).   Running iteratively means that any session job submitted within the interactive timeout, also known as interactive time to live (ITTL), is processed immediately. This allows some time for variational algorithms, such as VQE, to perform classical post-processing.

Reviewer 2: ```suggestion
Sessions run _iteratively_. Running iteratively means that any session job submitted within the interactive timeout, also known as interactive time to live (ITTL), is processed immediately. This allows some time for variational algorithms, such as VQE, to perform classical post-processing. To run jobs in batch mode, see [batch](run-primitives-batch).
```
It feels a little odd to introduce the idea of iteration then jump to batching then go back to explaining iteration, do i'd recommend rephrase like this
